### PR TITLE
Update catalog.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Added Response of the Category creation.

## Purpose of this pull request

This pull request (PR) add the Response of the Category creation. After Hit Request payload, Response data will be displayed in the post.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/rest/modules/catalog.html
https://devdocs.magento.com/guides/v2.4/rest/modules/catalog.html

### CODE REVIEW

File: src/guides/v2.3/rest/modules/catalog.md [hunk 1]
```diff
@@ -76,3 +78,59 @@ The following example uses the `POST V1/categories` call to assign four custom a
     }
 }
 ```
+
+## Response
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/modules/catalog.md:82]: Needs blank line after title

File: src/guides/v2.3/rest/modules/catalog.md [hunk 2]
```diff
@@ -76,3 +78,59 @@ The following example uses the `POST V1/categories` call to assign four custom a
...
+    "children": "",
+    "created_at": "2020-12-17 10:38:01",
+    "updated_at": "2020-12-17 10:38:01",
+    "path": "1/2/42",
+    "available_sort_by": [],
+    "include_in_menu": true,
+    "custom_attributes": [
+        {
+            "attribute_code": "description",
+            "value": "Women category description"
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/modules/catalog.md:100]:
```suggestion
            "value": "Women's category description"
```

File: src/guides/v2.3/rest/modules/catalog.md [hunk 3]
```diff
@@ -76,3 +78,59 @@ The following example uses the `POST V1/categories` call to assign four custom a
...
+        },
+        {
+            "attribute_code": "meta_title",
+            "value": "Women meta title"
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/modules/catalog.md:104]:
```suggestion
            "value": "Women's meta title"
```

File: src/guides/v2.3/rest/modules/catalog.md [hunk 4]
```diff
@@ -76,3 +78,59 @@ The following example uses the `POST V1/categories` call to assign four custom a
...
+        },
+        {
+            "attribute_code": "meta_keywords",
+            "value": "Women meta keywords"
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/modules/catalog.md:108]:
```suggestion
            "value": "Women's meta keywords"
```

File: src/guides/v2.3/rest/modules/catalog.md [hunk 5]
```diff
@@ -76,3 +78,59 @@ The following example uses the `POST V1/categories` call to assign four custom a
...
+        },
+        {
+            "attribute_code": "meta_description",
+            "value": "Women meta description"
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/modules/catalog.md:112]:
```suggestion
            "value": "Women's meta description"
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few small suggestions

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have updated the changes.

**Comment by keharper** [comment]
@RakeshJesadiya I think the example is poor, and changing "Women" to "Women's" isn't going to fix it.

Let's look at the example is trying to demonstrate. It's trying to create a new top-level category that looks suspiciously similar to the Women category in Luma. Luma doesn't need another "Women" or "Women's" category. But perhaps Luma could use a category called "Kids".

Here's the output of  `GET /V1/categories/20`, where 20 is the ID of the "Women" category.

```
{
  "id": 20,
  "parent_id": 2,
  "name": "Women",
  "is_active": true,
  "position": 2,
  "level": 2,
  "children": "21,22",
  "created_at": "2020-12-14 19:11:33",
  "updated_at": "2020-12-14 19:11:33",
  "path": "1/2/20",
  "available_sort_by": [],
  "include_in_menu": true,
  "custom_attributes": [
    {
      "attribute_code": "display_mode",
      "value": "PAGE"
    },
    {
      "attribute_code": "is_anchor",
      "value": "0"
    },
    {
      "attribute_code": "path",
      "value": "1/2/20"
    },
    {
      "attribute_code": "children_count",
      "value": "8"
    },
    {
      "attribute_code": "url_key",
      "value": "women"
    },
    {
      "attribute_code": "url_path",
      "value": "women"
    }
  ]
}
```

Create a new example that defines a new category. Use the output above a source of inspiration. You probably want to include the `url_key` and `url_path` attribute codes. Maybe you can think of a few more that would be interesting. Make the example look real.

**Comment by RakeshJesadiya** [comment]
@keharper I have updated the changes and add new category called Kids.

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**